### PR TITLE
fix(pr-review): post one line of why, then the change

### PR DESCRIPTION
**Pull request by steph-dove**
## Summary

Add-to-PR was posting too much: a two-sentence reason, a literal `Suggested change:` label, and the suggestion block. An inline comment should be one line saying what's wrong, then the change. The label also duplicated the header GitHub already prints above a suggestion block.

## Changes

- `findingCommentBody()` now returns one line of why, a blank line, then the suggestion. No label of ours.
- `findingWhyText()` takes the first sentence rather than the first two, and collapses newlines, so a wrapped paragraph lands as a single line instead of three.
- Severity, category and location still render from the card's own fields and never post, unchanged.
- Copy still sends exactly what Add-to-PR posts; both go through `findingCommentBody`.

Behavior before and after, for a finding citing a swallowed 429:

**Before**

```
The retry loop eats the 429, so a rate-limited call comes back looking fine. It only shows up under load.

Suggested change:
```suggestion
    if (attempt === last) throw err;
```
```

**After**

```
The retry loop eats the 429, so a rate-limited call comes back looking fine.

```suggestion
    if (attempt === last) throw err;
```
```

Edge cases kept: a why with no suggestion posts the why alone, a bare suggestion posts alone, and a finding with no label posts unchanged.

## Test Plan

- [x] Tests pass locally — 485 passed, 0 failed on this branch; eslint clean
- [x] Manually verified in the dev app

Covered by `test/util/pr-review-anchoring.test.js`:

- posted body is one line of why, then the change, unlabeled
- why is one sentence, on one line (including a wrapped paragraph)
- a finding with why but no suggestion posts the why alone
- a suggestion with no prose above it posts just the suggestion
- a finding with no suggestion label posts its text unchanged

I dropped a test I had added that stubbed `navigator.clipboard` to assert Copy matches Add-to-PR. Node 21+ makes `navigator` a read-only global, so the stub never applied and the test was exercising Node's globals rather than our logic. Both call sites share the `findingCommentBody` seam, which is what actually guarantees the match.

## Checklist

- [x] Code follows project conventions
- [x] No unrelated changes
- [x] Tests added/updated for new functionality
- [ ] Documentation updated if needed

## Note for review

Our label is gone, but GitHub prints its own "Suggested change" header above any ` ```suggestion ` fence — that's the applyable-suggestion widget, and `review-prompts.js:56` asks the model to produce exactly that. If the goal is no "suggested change" wording anywhere, the fence has to become a plain ` ``` ` block, which gives up the one-click Apply button. Not done here; it's a separate call.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: renderer/pr-review-implement.js [hunk 1]
```diff
@@ -335,21 +336,22 @@
...
     var suggestion = PR.findingSuggestionText(f);
-    if (!suggestion) return why;
     // No label in the text at all means findingSuggestionText returned the
     // whole body, which the why would duplicate.
     if (!/(?:^|\n)[ \t]*Suggested change:/i.test((f && f.text) || '')) return suggestion;
+    var why = PR.findingWhyText(f);
     if (!why) return suggestion;
-    return why + '\n\nSuggested change:\n' + suggestion;
+    if (!suggestion) return why;
+    return why + '\n\n' + suggestion;
```
Comment (steph-dove) [inline renderer/pr-review-implement.js]: Only a fence (line 226) and passes prose fixes straight through unfenced (line 228), so both now post as an unlabeled second paragraph.

Keep the label except when the suggestion already opens a ```suggestion fence, e.g. `var labelled = /^```suggestion\b/.test(suggestion) ? suggestion : 'Suggested change:\n' + suggestion;` then return `why + '\n\n' + labelled`.

File: renderer/pr-review-implement.js [hunk 2]
```diff
@@ -325,8 +325,9 @@
     return m ? text.slice(m.index + m[0].length).trim() : text.trim();
   };
 
-  // The prose above the "Suggested change:" label, capped at two sentences: the
-  // "why" that leads the posted comment, not the whole finding.
+  // One line saying what's wrong, taken from the prose above the "Suggested
+  // change:" label. First sentence only, with newlines collapsed, so a wrapped
+  // paragraph still posts as a single line.
```
Comment (steph-dove) [inline renderer/pr-review-implement.js:334]: main/state/review-prompts.js:55 still tells the model the body must open with the reason "in one or two sentences", so the model keeps writing two and this silently eats the second one.

Change review-prompts.js:55 to ask for a single sentence that names both what breaks and when, so the model puts everything in sentence one instead of losing the tail here.

File: renderer/pr-review-implement.js [hunk 3]
```diff
@@ -335,21 +336,22 @@
       .replace(/^[ \t]*\*\*[^\n]*\*\*[ \t]*$/gm, '')  // severity/category/location line
       .trim();
     if (!prose) return '';
-    var sentences = prose.split(/(?<=[.!?])\s+/).filter(function (s) { return s.trim(); });
-    return sentences.slice(0, 2).join(' ').trim();
+    var first = prose.split(/(?<=[.!?])\s+/).filter(function (s) { return s.trim(); })[0];
```
Comment (steph-dove) [inline renderer/pr-review-implement.js]: The split fires on any `.!?` followed by whitespace, so a first sentence containing an abbreviation truncates mid-thought.

Reject a break when the token before the period is a known abbreviation or a single letter, e.g. split on `/(?<=[.!?])\s+(?=[A-Z"'`(])/` and skip candidates where the preceding word matches `/\b(e\.g|i\.e|vs|etc|cf)\.$/i`.
